classes/form/CustomerForm.php

```diff
@@ -146,13 +147,34 @@ public function validate()
         }
 
         $passwordField = $this->getField('password');
-        if ((!empty($passwordField->getValue()) || $this->passwordRequired)
-            && Validate::isPlaintextPassword($passwordField->getValue()) === false) {
-            $passwordField->addError($this->translator->trans(
-                'Password must be between 5 and 72 characters long',
-                [],
-                'Shop.Notifications.Error'
-            ));
+        if (!empty($passwordField->getValue()) || $this->passwordRequired) {
+            if (Validate::isAcceptablePasswordLength($passwordField->getValue()) === false) {
+                $passwordField->addError($this->translator->trans(
+                    'Password must be between %d and %d characters long',
+                    [
+                        Configuration::get(PasswordPolicyConfiguration::CONFIGURATION_MINIMUM_LENGTH),
+                        Configuration::get(PasswordPolicyConfiguration::CONFIGURATION_MAXIMUM_LENGTH),
+                    ],
+                    'Shop.Notifications.Error'
+                ));
+            }
+
+            if (Validate::isAcceptablePasswordScore($passwordField->getValue()) === false) {
+                $wordingsForScore = [
+                    $this->translator->trans('Very weak'),
+                    $this->translator->trans('Weak'),
+                    $this->translator->trans('Average'),
+                    $this->translator->trans('Strong'),
+                    $this->translator->trans('Very strong'),
+                ];
+                $passwordField->addError($this->translator->trans(
+                    'The minimum score must be: %s',
```

Comment:
```suggestion
                    'The minimum score must be: %d',
```

Reply:
Not it's %s : because we set strings defined in the previous array.